src/snowflake/snowpark/functions.py

```diff
@@ -10744,13 +10745,15 @@ def make_interval(
 def snowflake_cortex_summarize(text: ColumnOrLiteralStr):
     """
     Summarizes the given English-language input text.
-
     Args:
         text: A string containing the English text from which a summary should be generated.
-
     Returns:
         A string containing a summary of the original text.
     """
+    warning(
+        "snowflake_cortex_summarize",
+        "snowflake_cortex_summarize is deprecated. Please consider using snowflake.cortex.Summarize instead.",
```

Comment:
better telling users where to get the method
```suggestion
        "snowflake_cortex_summarize is deprecated. Please consider installing snowflake-ml-python and using snowflake.cortex.Summarize instead.",
```